homework/index.js

```diff
@@ -30,18 +30,70 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
+  function createLayout() {
+    const root = document.getElementById('root');
+    createAndAppend('select', root, { id: 'repo-select' });
+    createAndAppend('div', root, { id: 'body-container' });
+    createAndAppend('div', document.getElementById('body-container'), { id: 'repo-details' });
```

Comment:
Instead of getting the body-container from the dom you can do something like this:

```javascript
const bodyContainer = createAndAppend('div', root, { id: 'body-container' });
createAndAppend('div', bodyContainer, { id: 'repo-details' });
```

Both will work the same but using the variable will be faster (doesn't matter in the homework but important when working)